ChangeLog

```diff
@@ -0,0 +1,2 @@
+CHANGES
+=======
```

Comment:
Are you going to add content here from the [past 10 releases](https://github.com/CloverHealth/temporal-sqlalchemy/releases)?

Reply:
I GUESS